src/gui/torrentcontentmodelfile.cpp

```diff
@@ -33,42 +33,48 @@
 
 TorrentContentModelFile::TorrentContentModelFile(const QString &fileName, qulonglong fileSize,
                                                  TorrentContentModelFolder* parent, int file_index)
-  : TorrentContentModelItem(parent)
-  , m_fileIndex(file_index)
+    : TorrentContentModelItem(parent)
+    , m_fileIndex(file_index)
 {
-  Q_ASSERT(parent);
+    Q_ASSERT(parent);
 
-  m_name = fileName;
+    m_name = fileName;
 
-  // Do not display incomplete extensions
-  if (m_name.endsWith(".!qB"))
-    m_name.chop(4);
+    // Do not display incomplete extensions
+    if (m_name.endsWith(".!qB"))
```

Comment:
And it is *not* used in 11 places. I would better do a separate PR eliminating all of those.